packages/cdc-react/src/components/Sidebar/Sidebar.test.tsx

```diff
@@ -44,3 +45,44 @@ describe("Sidebar component", () => {
     expect(screen.queryByText("Logout")).not.toBeInTheDocument();
   });
 });
+
+describe("Responsiveness", () => {
+  describe("Media Query Breakpoint Mocks", () => {
+    const mockScreenSize = (size: ScreenSize) => {
```

Comment:
I don't believe this function is needed.  Since we aren't passing `window` into this function, or `resizeView`, the `resizeView` has a hidden side effect.  I would prefer to not obfuscate that in another wrapper function.  In addition, we might want to have a test that renders a sidebar with different props